.github/actions/load-docker-image/action.yml

```diff
@@ -0,0 +1,52 @@
+name: load-docker-image
+decription: |
+  Pull a docker image and cache it.
+
+inputs:
+  image:
+    required: true
+    type: string
+  cache_dir:
+    required: false
+    type: string
+    default: "~/.docker-images-cache"
```

Comment:
Let's only use GHA-defined variables instead of `~`; the list is: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables.

Reply:
Good call!